pkg/controller/infinispan/resources/service/xsite_service.go

```diff
@@ -0,0 +1,65 @@
+package service
+
+import (
+	ispnv1 "github.com/infinispan/infinispan-operator/pkg/apis/infinispan/v1"
+	consts "github.com/infinispan/infinispan-operator/pkg/controller/constants"
+	"github.com/infinispan/infinispan-operator/pkg/controller/infinispan"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func computeSiteService(ispn *ispnv1.Infinispan) *corev1.Service {
```

Comment:
Should we just incorporate this as part of `server_controller.go` as it's only a singe method?

Reply:
Yes, thanks, will fix